common/network/NetworkUtils.cpp

```diff
@@ -160,11 +164,11 @@ uint32_t NetworkToHost(uint32_t value) {
 }
 
 uint64_t NetworkToHost(uint64_t value) {
-#ifdef HAVE_ENDIAN_H
+#if defined(HAVE_ENDIAN_H) || defined(__APPLE__)
   return be64toh(value);
 #else
 #error "No be64toh for NetworkToHost, please report this."
-#endif  // HAVE_ENDIAN_H
+#endif  // defined(HAVE_ENDIAN_H) || defined(__APPLE__)
```

Comment:
Then switch this, and the following cases, to something more like https://github.com/OpenLightingProject/ola/blob/e8c755cdb1aba608785cce501ff5cc86f866e6d6/common/network/NetworkUtils.cpp#L684-L712 (but I don't think we need the extra wrapper functions here).